File: server/Model/Personal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class Personal
    {
        public string first_name { get; set; }
        public string last_name { get; set; }
        public long id { get; set; }
        public string address { get; set; }
        // public string city { get; set; }
        public DateTime birthday { get; set; }
        public long phone { get; set; }
        public long mobile { get; set; }
    }
}

File: client/src/app/models/Personal.ts
import { KeyObject } from "crypto";
import { Corona_vaccine } from "./Corona_vaccine";
import { IllnessDate } from "./IllnessDate";
import { Vaccine_company } from "./Vaccine_company";

export class Personal{
   
    first_name:string | undefined;
    last_name!: string;
    id!: number;
    address!: string;
    //city!: string;
    birthday!: Date;
    phone!: number;
    mobile!: number;
}